css/nav.css

```diff
@@ -1,21 +1,233 @@
+    position: absolute;
+    top: 25%;
+    left: 25%;
+    width: 50%;
+    height: 50%;
+}
+
+#close-nav-menu {
+    position: relative;
+    width: 2em;
+    height: 2em;
+    padding: 0;
+    margin-top: 1em;
+    border-radius: 50%;
+    background: transparent;
+    border: 2px solid white;
+    
+    cursor: pointer;
+}
+
+#close-nav-menu img {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    left: 25%;
+    top: 25%;
+}
+
+@media only screen and (max-width: 1000px) {
+    /* portrait / small width styling, show burger menu */
+    #nav-button {
+        display: block;
+    }
+
+    nav {
+        position: absolute;
+        top: 0;
+        right: 0;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        width: 100vw;
+        height: 100vh;
+        z-index: 999;
+
+        background-color: rgb(59 71 179 / 80%);
+
+        visibility: hidden;
+        transition: 0.25s opacity ease;
+        opacity: 0;
+
+        pointer-events: none;
+    }
+
+    nav > * {
+        transition: 0.5s transform ease, 0.5s opacity ease;
+        transform: translateY(-2em);
+        opacity: 0;
+    }
+
+    nav a {
+        background-color: transparent !important;
+        padding: 0.5em 0;
+        margin: 0.5em 0;
+    }
+
+    nav a.active {
+        transform: translateY(-50%) scale(1.1);
+        border-bottom: 3px solid white;
+        border-bottom-style: double;
+    }
+
+    nav a:hover:not(.active),
+    nav button:hover:not(.active) {
+        transition-duration: 0.25s;
+        transform: scale(1.1);
+    }
+
+    nav.active {
+        visibility: visible;
+    }
+
+    nav.open {
+        opacity: 1;
+        pointer-events: auto;
+    }
+
+    nav.open > * {
+        transform: translateY(0%);
+        opacity: 1;
+    }
+
+    nav.open a.active {
+        transform: translateY(0%) scale(1.1);
+        opacity: 1;
+    }
+}
+
+@media only screen and (min-width: 1000px) {
+    /* wide styling, for when the tabs are all visible on screen */
+    nav a#nav-home {
+        flex-grow: 1;
+    }
+    
+    nav a#nav-home span {
+        visibility: hidden;
+        width: 0;
+    }
+
+    nav a#nav-home img {
+        width: 5em;
+        height: 5em;
+        margin-bottom: 0.5em;
+    }
+
+    nav a:not(#nav-home) {
+        padding: 0.55rem 2.7rem;
+        transform-origin: 50% 100%;
+        font-size: 0.6em;
+    }
+
+    nav a:not(#nav-home).active {
+        font-size: 0.75em;
+    }
+
+    /* Setting tab styling to ::before so we can display tab outline w/ keyboard */
+    nav a:not(#nav-home)::before {
+        content: "";
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        clip-path: url(#navClip);
+
+        z-index: -1; /* render underneath text */
+    }
+
+    nav a:first-of-type {
+        transform-origin: 0 100%;
+    }
+
+    /* specifying #nav-credits here since the last element in the nav is not a tab */
```

Comment:
could you explain this more? what does "not a tab" mean? I didn't read the code super closely and might have missed this

Reply:
yeah! Originally I had `:last-of-type` as the selector however the selector is not targeting the appropriate tab (`#nav-credits` in this case) because it's not the last child element in `nav`. The last child element is the close button that we need in the hamburger menu.